scripts/genebuild/create_annotation_configs.pl

```diff
@@ -96,9 +96,9 @@
   -dbname  => 'ncbi_taxonomy');
 
 #Adding registry details to hash for populating main config
-$general_hash->{registry_host} = $assembly_registry_host;
-$general_hash->{registry_port} = $assembly_registry_port;
-$general_hash->{registry_db} = $assembly_registry->{_dbc}->{_dbname};
+#$general_hash->{registry_host} = $assembly_registry_host;
+#$general_hash->{registry_port} = $assembly_registry_port;
+#$general_hash->{registry_db} = $assembly_registry->{_dbc}->{_dbname};
```

Comment:
The commented code must be removed because it doesn't explain the code

Reply:
Changes made as requested